controller/config.go

```diff
@@ -300,6 +308,14 @@ const (
 	// batches we will process. (deprecated)
 	DefaultMaxPruneTxnPasses = 100
 
+	// DefaultAgentLogfileMaxSize is the maximum file size in MB of each
+	// agent/controller log file.
+	DefaultAgentLogfileMaxSize = 300
```

Comment:
I think the default here could be a little lower, like 100. The primary source of logs is still debug-log which goes straight to mongo.

Reply:
Done, thanks.